contracts/TimedAssetLockContract/Deployment.py

```diff
@@ -2,12 +2,29 @@
 from algosdk import account, mnemonic, logic, encoding
 from akita_inu_asa_utils import *
 from pyteal import *
+from helpers import create_fakita_inu_signed_txn
 
 
 def deploy_app(client, private_key, approval_program, clear_program, global_schema, local_schema, app_args):
     # define sender as creator
     public_key = account.address_from_private_key(private_key)
 
+    # create ASA

```

Comment:
Think of the code within contracts/* to be code that we will be actually deploying with and not test code, for example, we will not be creating a new asa when we go to really deploy this contract